kubemarine/resources/schemas/definitions/plugins/local-path-provisioner.json

```diff
@@ -35,7 +35,7 @@
     },
     "helper-pod-image": {
       "type": "string",
-      "default": "library/busybox:1.35.0"
```

Comment:
I believe no need _library_ prefix for dockerhub

Reply:
done

Reply:
Put simply, the simplest solution would be to remove "default" for both "image" and "helper-pod-image"

Reply:
@dmyar21 pls remove

Reply:
done